### PR TITLE
[P4 tests] [Team 8] Add Tf-Idf Search tests

### PR DESCRIPTION
**Pull request by NinoXing**
What's your team number?
Team 8

What is the functionality being tested?
Search Tf-Idf

Describe your tests briefly:
test1 tests if the function works well when topk is null.
test2 tests if the function works well with normal topk number.

Does each test case have comments/documentation?
Yes

### CODE REVIEW

File: src/test/java/edu/uci/ics/cs221/index/ranking/Team8TfIdfTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    /**
+     Test if searchTfIdf function works well with topK == null
+     **/
+    @Test
+    public void test1(){
+        List<String> keywords = Arrays.asList("apple", "apple", "rotten");
+        Iterator<Pair<Document, Double>> res = indexmanger.searchTfIdf(keywords,null);
+        List<Document> resDoc = Arrays.asList(documents[1],documents[2],documents[0],documents[4]);
```
Comment (DanniUCI) [inline src/test/java/edu/uci/ics/cs221/index/ranking/Team8TfIdfTest.java]: Review from Team 9
Your test case looks good. But here is a typo. We think documents[4] should change to documents[3].

**Comment by DanniUCI** [comment]
Review from Team 9
Your test case looks good. But there is a typo at line 57, we think documents[4] should be replaced by documents[3].

**Comment by NinoXing** [comment]
> Review from Team 9
> Your test case looks good. But there is a typo at line 57, we think documents[4] should be replaced by documents[3].

Thanks a lot~ My bad, I have corrected it.

**Comment by adslchen** [comment]
Review from Team22
Your test case looks good. Well-designed.

**Pull request merged**